src/styles/variables.scss

```diff
@@ -1,11 +1,14 @@
-$active :#B3FFBF;
-$gray : #F5F5F5;
-$darkgray : #808080;
-$black: #202020;
-$outline: #B2FFBF;
+:root {
+    $active : #B3FFBF;
+    $gray : #F5F5F5;
+    $darkgray : #808080;
+    $black: #202020;
+    $outline: #B2FFBF;
+    --outline: #B2FFBF;
```

Comment:
I added two variables for now 'cus Idk if other components can use "--black" if "$black" is not available